README.md

```diff
@@ -783,6 +956,12 @@ In such cases, you can generate sample user accounts using the steps below. This
     chmod +x insert_sample_data.sh && \
     ./insert_sample_data.sh
     ```
+-   **Windows**
```

Comment:
@MallanagoudaB adding this here for 12.  **Start The Portal** section. correct the indentation of the section .  make sure that after creating  user account only we ask user to explore the portal